src/pages/docs/FAQs/general/faqs-on-classic-and-modern-engines.md

```
@@ -0,0 +1,65 @@
+page_id: "faqs-on-classic-and-modern-engines"
+search_keyword: ""
+warning: false
+---
+
+---
+
+Following are some frequently asked questions about Testsigma's Classic and Modern execution engines. The engine you choose when creating an application determines how your tests run, which add-ons you can use, and which features are available to you. Use this guide to find answers about how the two engines differ, when to pick one over the other, and what stays the same for your existing applications.
+
+---
+
+### **1. Can I change the execution engine after creating an application?**
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)